core/plugins/org.fusesource.ide.foundation.core/src/org/fusesource/ide/foundation/core/xml/namespace/FindNamespaceHandlerSupport.java

```diff
@@ -69,23 +67,20 @@ public boolean parseContents(InputSource contents) throws IOException, ParserCon
 		return true;
 	}
 
-	private final SAXParser createParser(SAXParserFactory parserFactory) throws ParserConfigurationException, SAXException,
-			SAXNotRecognizedException, SAXNotSupportedException {
-				parserFactory.setNamespaceAware(true);
-				final SAXParser parser = parserFactory.newSAXParser();
-				final XMLReader reader = parser.getXMLReader();
-				//reader.setProperty("http://xml.org/sax/properties/lexical-handler", this); //$NON-NLS-1$
-				// disable DTD validation (bug 63625)
-				try {
-					//	be sure validation is "off" or the feature to ignore DTD's will not apply
-					reader.setFeature("http://xml.org/sax/features/validation", false); //$NON-NLS-1$
-					reader.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false); //$NON-NLS-1$
-				} catch (SAXNotRecognizedException e) {
-					// not a big deal if the parser does not recognize the features
-				} catch (SAXNotSupportedException e) {
-					// not a big deal if the parser does not support the features
-				}
-				return parser;
-			}
+	private final SAXParser createParser(SAXParserFactory parserFactory) throws ParserConfigurationException, SAXException {
+		parserFactory.setNamespaceAware(true);
+		final SAXParser parser = parserFactory.newSAXParser();
+		final XMLReader reader = parser.getXMLReader();
+		//reader.setProperty("http://xml.org/sax/properties/lexical-handler", this); //$NON-NLS-1$
+		// disable DTD validation (bug 63625)
+		try {
+			//	be sure validation is "off" or the feature to ignore DTD's will not apply
+			reader.setFeature("http://xml.org/sax/features/validation", false); //$NON-NLS-1$
+			reader.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false); //$NON-NLS-1$
+		} catch (SAXNotRecognizedException | SAXNotSupportedException e) {
+			// not a big deal if the parser does not recognize the features
```

Comment:
I am not a friend of ignoring and not logging exceptions

Reply:
logging now at this old code place

Reply:
I know its old code and not touched in this PR but I think its good to improve it.